src/components/bounty-setup/BountySetup.svelte

```diff
@@ -1,12 +1,13 @@
 <script context="module">
-	export const TABS = ['Bounty Creation', 'Referendum', 'Curator Proposal'];
+	export const TABS = ['Bounty Creation', 'Approval', 'Curator Proposal'];
 	export type BountyTab = (typeof TABS)[number];
 </script>
 
 <script lang="ts">
 	import BountyCreation from './BountyCreation.svelte';
 	import BountySetupTab from './BountySetupTab.svelte';
-	let activeTab = TABS[2];
+	import ApprovalReferendum from './ApprovalReferendum.svelte';
```

Comment:
```suggestion
	import ApprovalReferendum from './ApprovalReferendum.svelte';
	
```
Nit: It's nice to separate the imports and the code with a new line.